src/security/tests/cli_security_tests.c

```diff
@@ -301,7 +301,7 @@ test_request_credentials_connects_to_env_socket(void **state)
 
```

Comment:
Need to update the check above - the sockaddr string passed in should be essentially getenv_return + "/agent.sock"
Could update the getenv_return value, too, to make clear we expect it to be a directory now.